src/examples/PopupExamplePage.tsx

```diff
@@ -14,7 +21,49 @@ export const PopupExamplePage: React.FunctionComponent<{}> = () => {
   const [showPopup3, setShowPopup3] = useState(false);
   const [showPopup4, setShowPopup4] = useState(false);
 
-  const myRef = useRef();
+  const popupRef: React.RefObject<Popup> = React.createRef<Popup>();
+  const popup2Ref: React.RefObject<Popup> = React.createRef<Popup>();
+  const popup3Ref: React.RefObject<Popup> = React.createRef<Popup>();
+  const popup4Ref: React.RefObject<Popup> = React.createRef<Popup>();
+
+  const viewRef: React.RefObject<View> = React.createRef<View>();
+  const view2Ref: React.RefObject<View> = React.createRef<View>();
+  const view3Ref: React.RefObject<View> = React.createRef<View>();
+  const view4Ref: React.RefObject<View> = React.createRef<View>();
+
+  useEffect(() => {
+    setTimeout(setFocus, 50);
+  });
+
+  const setFocus = () => {
+    if (popupRef.current?.props.isOpen) {
+      const reactTag = findNodeHandle(viewRef.current);
```

Comment:
Can we simplify some more and stop calling findNodeHandle?   That method is going away in fabric, and I'm not sure why you need to use it here.

Reply:
Removed the `findNodeHandle` calls